hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -378,6 +378,14 @@ private FlinkOptions() {
       .withDescription("Enables data-skipping allowing queries to leverage indexes to reduce the search space by "
           + "skipping over files");
 
+  @AdvancedConfig
+  public static final ConfigOption<Boolean> READ_PARTITION_DATA_SKIPPING_ENABLED = ConfigOptions
+      .key("read.partition.data.skipping.enabled")
```

Comment:
This option can be eliminated, when data skipping is enabled, we always enable partiton stats pruning if the partition metadata is there.

Reply:
We already have `read.data.skipping.enabled`, I don't think there is necessity to add more specific options.